commons/src/main/java/com/powsybl/commons/extensions/ExtensionXmlSerializer.java

```diff
@@ -6,28 +6,47 @@
  */
 package com.powsybl.commons.extensions;
 
+import com.powsybl.commons.Versionable;
 import com.powsybl.commons.xml.XmlReaderContext;
 import com.powsybl.commons.xml.XmlWriterContext;
 
 import javax.xml.stream.XMLStreamException;
 import java.io.InputStream;
+import java.util.Collections;
+import java.util.List;
 
 /**
  * An ExtensionProvider able to serialize/deserialize extensions from XML.
  *
  * @author Mathieu Bague <mathieu.bague at rte-france.com>
  */
-public interface ExtensionXmlSerializer<T extends Extendable, E extends Extension<T>> extends ExtensionProvider<T, E> {
+public interface ExtensionXmlSerializer<T extends Extendable, E extends Extension<T>> extends ExtensionProvider<T, E>, Versionable {
```

Comment:
Please, could you add javadoc to explain the expected behaviour, specially for method such as `getXsdAsStream()` that should return the latest (current) XSD